_episodes/06-control-flow.md

```diff
@@ -82,6 +82,75 @@ end
 
 Then she goes about implementing the `AbstractArray` interface.
 
+> ## Did if subtyping work?
+>
+> Now `Trebuchet` was made a subtype of the `AbstractArray` interface. We should be able to check this in the julia REPL.
+> How would you do that?
+>
+> 1. `Trebuchet` appears in the output of `subtypes(AbstractArray)`
+> 2. `AbstractArray` appears in the output of `supertype(Trebuchet)`
+> 3. `Trebuchet` appears in the output of `subtype(AbstractArray)`
+> 4. `AbstractArray` appears in the output of `supertypes(Trebuchet`
+>
```

Comment:
Maybe add the use of `isa`, eg `my_trebuchet isa Trebuchet` and `isa(my_trebuchet, AbstractArray)`

Reply:
Good point, this could replace one of the "correct" answers, I think.

Reply:
on the other hand, this would require an instance of `Trebuchet`, right. So maybe it doesn't fit into this exercise. Maybe another new one?

Reply:
> on the other hand, this would require an instance of Trebuchet, right

Yes, exactly. Happy to defer to you if you think it doesn't fit.